packages/services/mock-data-service/src/main/java/software/uncharted/terarium/mockdataservice/models/Project.java

```diff
@@ -0,0 +1,33 @@
+package software.uncharted.terarium.mockdataservice.models;
```

Comment:
Projects look like this: 
```
{
  "name": "string",
  "description": "string",
  "assets": {
    "resource-type": "list-of-ordered-ints"
  },
  "status": "string"
}
```

It has the nested assets in the object

Reply:
fixed